cosim/black-parrot-example/vivado/Makefile

```diff
@@ -7,9 +7,8 @@ BASE_FLIST ?= $(abspath ../flist.vcs)
 BP_FLIST ?= $(abspath flist.blackparrot.vcs)
 FLIST ?= $(abspath flist.vcs)
 
-$(BP_FLIST): $(BLACKPARROT_DIR)/bp_top/syn/flist.vcs
-	cp $^ $@
-	sed -i "s/BASEJUMP_STL_DIR/BP_BASEJUMP_STL_DIR/g" $@
+$(BP_FLIST): $(BLACKPARROT_DIR)/bp_top/syn/flist.vcs $(BASE_FLIST)
+	cat $^ | sed "s/BASEJUMP_STL_DIR/BP_BASEJUMP_STL_DIR/g" | envsubst > $@
```

Comment:
What happens if basejump in import/ and basejump in the BP RTL repo get out of sync? Does that cause issues with having two different implementations of the same module?

Reply:
Yeah, it theoretically would. My logic is that BP is more likely to use newer versions of modules than ZP, but really there's not a great solution